src/content/changes/highlights/2021-06-25.md

```diff
@@ -0,0 +1,21 @@
+---
+date: 2021-06-25T10:00:00
+title: Highlights for the week ending Jun 25, 2021
+changes_categories:
+- Highlights
+owner:
+- https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+As we aim to align our terminology with the Kubernetes project where possible, we started replacing the term _master node_ with _control plane node_ in our communication and user interfaces.
+
+## Managed apps
+
+[EFK v0.5.3](https://docs.giantswarm.io/changes/managed-apps/efk-stack-app/v0.5.3/) adds support to override the Kibana image registry, increasing flexibility. For example, allowing customers to install their own plugins, set default configs, etc.
```

Comment:
@QuentinBisson , @giantswarm/app-squad-efk - please check if OK w you :)

Reply:
Good for me